shell/browser/api/electron_api_app_mac.mm

```diff
@@ -58,6 +58,15 @@
   [NSApp setActivationPolicy:activation_policy];
 }
 
+bool App::IsRunningUnderRosettaTranslation() {
+  int ret = 0;
+  size_t size = sizeof(ret);
+  if (sysctlbyname("sysctl.proc_translated", &ret, &size, NULL, 0) == -1) {
```

Comment:
`NULL` -> `nullptr`?

Reply:
Sticking with apples sample code here, makes for easier grep / replace if their recommendation changes in the future

Reply:
You're going to grep on NULL rather than sysctl.proc_translated? :smile:

This line will break grep anyway if you change `ret` to `cur` or `curval`